pkg/server/controllers.go

```diff
@@ -1724,6 +1726,7 @@ func (s *Server) installCacheController(ctx context.Context, config *rest.Config
 		s.DiscoveringDynamicSharedInformerFactory,
 		s.CacheKcpSharedInformerFactory,
 		cachedResourceInformer,
+		cachedResourceEndpointSliceInformer,
```

Comment:
just do inline:
`s.KcpSharedInformerFactory.Cache().V1alpha1().CachedResourceEndpointSlices()`

Reply:
I think one above was my mistake :)

Reply:
It bit strange now that namespaces and secrets are inline, other not :)

Reply:
> I think one above was my mistake :)

That one is actually required because it's used in the returned wrapper below

Reply:
It's just that I added a `HasSynced()` call into the wait function below.